annotated.go

```diff
@@ -381,9 +381,15 @@ func (ann *annotated) parameters() (
 			Type: t,
 		}
 
+		var paramTag string
 		if i < len(ann.ParamTags) {
-			field.Tag = reflect.StructTag(ann.ParamTags[i])
+			paramTag = ann.ParamTags[i]
 		}
+		if i == ft.NumIn()-1 && ft.IsVariadic() && !strings.Contains(paramTag, "optional:") &&
+			!strings.Contains(paramTag, "group:") {
+			paramTag = paramTag + " optional:\"true\""
+		}
```

Comment:
I feel like the string matching here will be fragile. Normally, since the format of struct tags is structured, we would talk about parsing it and checking, but that might be moot because:

Based on the issue description, we want to add `optional:"true"` only if the variadic parameter has no other tags. So the `[]Foo` for `whatever(...Foo)` defaults to optional only if the user didn't explicitly specify anything for it.
So the check we actually need is: param tags for this argument is empty, or is too short to hit this argument.

Reply:
Ping. This comment is still valid.